deis-logger-beta1-test/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: deis-logger-beta1-test
+home: http://github.com/deis/deis
+version: 0.1.0
```

Comment:
I would change that.  Looks like the `workflow-beta1-test` chart uses a value of `v2-beta` here.